carbonmark-api/src/plugins/open-api.ts

```diff
@@ -1,142 +1,43 @@
-                  active: { type: "boolean" },
-                  deleted: { type: "boolean", items: [] },
-                  batches: { type: "array", items: [] },
-                  batchPrices: { type: "array" },
-                  singleUnitPrice: { type: "number" },
-                  project: {
-                    type: "object",
-                    properties: {
-                      name: { type: "string" },
-                      category: { type: "string" },
-                    },
-                  },
-                },
-              },
-            },
-            activities: {
-              type: "array",
-              items: {
-                type: "object",
-                properties: {
-                  id: { type: "number" },
-                  amount: { type: "number" },
-                  previousAmount: { type: "number" },
-                  price: { type: "number" },
-                  previousPrice: { type: "number" },
-                  timeStamp: { type: "number" },
-                  project: {
-                    type: "object",
-                    properties: {
-                      key: { type: "string" },
-                    },
-                  },
-                  seller: {
-                    type: "object",
-                    properties: {
-                      id: { type: "address" },
-                    },
-                  },
-                  buyer: {
-                    type: "object",
-                    properties: {
-                      id: { type: "address" },
-                    },
-                  },
-                },
-              },
-            },
-          },
-        },
-        Project: {
-          type: "object",
-          properties: {
-            id: { type: "number" },
-            key: { type: "string" },
-            projectID: { type: "string" },
-            name: { type: "string" },
-            methodology: { type: "string" },
-            vintage: { type: "number" },
-            projectAddress: { type: "address" },
-            registry: { type: "string" },
-          },
-        },
-        Category: {
-          type: "object",
-          properties: {
-            id: { type: "string" },
-          },
-        },
-        Country: {
-          type: "object",
-          properties: {
-            id: { type: "string" },
-          },
-        },
-      },
-      securityDefinitions: {
-        apiKey: {
-          type: "apiKey",
-          name: "apiKey",
-          in: "header",
-        },
-      },
+For a developer guides and example implementations, or to learn more about Carbonmark and Digital Carbon Market, view our product knowledge base at <a href="https://docs.carbonmark.com">docs.carbonmark.com</a>.
+## 
+`,
+    termsOfService: "https://www.carbonmark.com/blog/terms-of-use",
+    contact: {
+      name: "Support",
+      url: "https://share-eu1.hsforms.com/1RWJWvyrHT1C_an4cZOHH3gfhhlr",
```

Comment:
Nit: some of the urls already exist in the `lib/constants` file, for this one specifically it's `carbonmarkContactForm`.

Reply:
Sorry, this should not be in this PR. I messed up while rebasing. I will force push.